mapclientplugins/filelocationsinkstep/qt/configuredialog.ui

```diff
@@ -57,23 +47,43 @@
         </property>
        </spacer>
       </item>
-      <item row="2" column="0">
-       <widget class="QLabel" name="label1">
+      <item row="0" column="2">
+       <widget class="QLineEdit" name="lineEdit0"/>
+      </item>
+      <item row="0" column="0">
+       <widget class="QLabel" name="label0">
         <property name="text">
-         <string>Output directory:  </string>
+         <string>identifier:</string>
         </property>
        </widget>
       </item>
+      <item row="2" column="2">
+       <layout class="QHBoxLayout" name="horizontalLayout">
+        <item>
+         <widget class="QLineEdit" name="lineEditFileLocation"/>
+        </item>
+        <item>
+         <widget class="QPushButton" name="pushButtonFileChooser">
```

Comment:
This is not for a file chooser and needs to be updated to show that is is for a directory.